test/controllers/EmailControllerSpec.scala

```diff
@@ -41,10 +41,10 @@ class EmailControllerSpec extends SpecBase {
       when(mockHttpClient.get(any)(any)).thenReturn(requestBuilder)
 
       running(app) {
-        val connector = app.injector.instanceOf[CustomsFinancialsApiConnector]
+        val connector = app.injector.instanceOf[DataStoreConnector]
 
-        val result: Future[Option[String]] = connector.isEmailUnverified(hc)
-        await(result) mustBe expectedResult
+        val result: Future[Option[EmailUnverifiedResponse]] = connector.retrieveUnverifiedEmail(hc)
```

Comment:
You'll have to update this to be as per below once method for `retrieveUnverifiedEmail` is changed as per comment

```scala
        val result: Future[Option[String]] = connector.retrieveUnverifiedEmail(hc)
        await(result) mustBe expectedResult
```